nixos/modules/programs/rust-motd.nix

```diff
@@ -28,9 +55,35 @@ in {
       '';
     };
     settings = mkOption {
-      type = types.submodule {
+      type = types.attrsOf (types.submodule {
         freeformType = format.type;
-      };
+        options.priority = mkOption {
+          type = types.int;
+          default = modules.defaultOrderPriority;
+          description = mdDoc ''
+            In `rust-motd`, the order of the sections in TOML correlates to the order
+            of the items displayed in the resulting `motd`. Since attributes in Nix are
+            ordered alphabetically, e.g. `banner` would always be before `uptime`.
```

Comment:
The message is clear. However, the “Since” made me expect another part of the sentence.

Reply:
Very good point. I guess I was getting tired when I wrote the explanations here, will fix that :)